test/libyul/yulOptimizerTests/unusedFunctionParameterPruner/multiple_return.yul

```diff
@@ -0,0 +1,24 @@
+{
+    let a, b, c := f(sload(0))
```

Comment:
Can you add such a test to the fullSuite so that we see that unused return variables are properly removed at the calling site?

Reply:
test/libyul/yulOptimizerTests/fullSuite/unusedFunctionParameterPruner_return.yul is a testcase for it.